services/orchestrator-agent/webhook_server.py

```
@@ -522,15 +601,36 @@ async def federation_status(request: web.Request) -> web.Response:
 
+    async def ready(request: web.Request) -> web.Response:
+        """Readiness probe – requires DB connectivity (distinct from liveness /health)."""
+        from db import get_pool as _get_pool
+
+        try:
+            pool = await _get_pool()
+            conn = await pool.acquire()
+            await conn.execute("SELECT 1")
+            await pool.release(conn)
+            return web.json_response({"status": "ready", "service": "orchestrator-agent"})
+        except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)